experimental/iterators/test/simple.mlir

```diff
@@ -0,0 +1,15 @@
+// TODO: this currently does not propertly connect to an actual execution
+
+// R-UN: mlir-opt %s -convert-std-to-llvm -reconcile-unrealized-casts | \
+// R-UN: mlir-cpu-runner -e main -entry-point-result=void \
+// R-UN:   -shared-libs=$(pwd)/lib/Utils/libruntime_utils.so \
+// R-UN: | FileCheck %s
```

Comment:
I have managed to run this manually. How is this normally run? I guess by some CMake infrastructure? I'd have to include this project into the parent build to achieve this, right? (I guess I'll have to do that sooner or later, anyways.)

Reply:
In LLVM proper, this is using [lit](https://llvm.org/docs/CommandGuide/lit.html) and [FileCheck](https://llvm.org/docs/CommandGuide/FileCheck.html).

@gysit connected these pieces in a python script here: https://github.com/google/iree-llvm-sandbox/blob/main/run_tests.py; you should be able to copy-paste and reuse and then s/R-UN/RUN in the example.